build/requirements.txt

```diff
@@ -5,3 +5,8 @@ gspread==0.2.5
 gunicorn==19.3.0
 oauth2client==1.4.12
 requests[security]==2.7.0
+
+# requirements for airtable to google groups sync
+#requests[security]==2.7.0
```

Comment:
Oh gotcha, yeah I suppose the "right" robust way to do this would to have a python virtualenv for each program and install just what they need into their own env.

I see what you mean here, so leaving it commented makes sense.